README.md

```diff
@@ -130,7 +130,9 @@ This plugin enables you to use the native macOS search to find files and folders
 
 This plugin enables you to browse through your local file system. To start you have to enter a valid absolute filepath.
 
-![file-browser](assets/feature-file-browser.png)
```

Comment:
Did you remove this image intentionally? If so, why?

Reply:
No, it was a mistake, thanks for mentioning